cmd/generate-workflow.go

```diff
@@ -64,7 +64,7 @@ func (gwc *generateWorkflowCmd) generateWorkflows() error {
 	if gwc.deployType == "" {
 		selection := &promptui.Select{
 			Label: "Select k8s Deployment Type",
-			Items: []string{"helm", "kustomize", "manifests"},
+			Items: []string{"helm", "kustomize", "manifest"},
```

Comment:
this is breaking, instead keep it as is and move the azure-pipeline to be "manifests"

https://github.com/Azure/draft/blob/43cdf6b7802fbdb8579ac83c13fb3b074381462e/cmd/info.go#L19C1-L23C2